plugins/inputs/nats_consumer/README.md

```diff
@@ -56,8 +56,13 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   # password = ""
 
   ## Optional NATS 2.0 and NATS NGS compatible user credentials
+  ## Expects a credentials file with a JWT and nkey seed
```

Comment:
This update makes it sound like this config option never worked before as it requires the nkey seed config option? Is that your intent?

Reply:
It worked, my intention was to explicitly state the difference between `nats.creds` authentication and nkey. 

The `nats.creds` option requires special configuration on the NATS server, whereas user+pass and nkey are very simple. And mixed authentication with `nats.creds` and the other types is possible, but also requires some extra configuration. 

I updated the comments in my latest commit for each type of auth and I added a note at the top to be extra clear.